addons/website_sale/views/product_views.xml

```diff
@@ -102,7 +102,7 @@
                             <field name="website_style_ids" widget="many2many_tags" groups="base.group_no_one"/>
                         </group>
                     </group>
-                    <group name="product_template_images" string="Extra Product Images">
+                    <group name="product_template_images" string="Extra Product Images / Videos">
```

Comment:
Ideally the name on the o2m on the product/template field should be renamed too (it is the title of the modal when adding a new video).